providers/dns/exoscale/exoscale.go

```diff
@@ -34,22 +38,22 @@ type Config struct {
 	APIKey             string
 	APISecret          string
 	Endpoint           string
-	HTTPClient         *http.Client
+	APIZone            string
+	HTTPTimeout        time.Duration
 	PropagationTimeout time.Duration
 	PollingInterval    time.Duration
-	TTL                int
+	TTL                int64
 }
 
 // NewDefaultConfig returns a default configuration for the DNSProvider.
 func NewDefaultConfig() *Config {
+	ttl := int64(env.GetOrDefaultInt(EnvTTL, dns01.DefaultTTL))
 	return &Config{
-		TTL:                env.GetOrDefaultInt(EnvTTL, dns01.DefaultTTL),
+		TTL:                ttl,
 		PropagationTimeout: env.GetOrDefaultSecond(EnvPropagationTimeout, dns01.DefaultPropagationTimeout),
 		PollingInterval:    env.GetOrDefaultSecond(EnvPollingInterval, dns01.DefaultPollingInterval),
-		HTTPClient: &http.Client{
-			Timeout:   env.GetOrDefaultSecond(EnvHTTPTimeout, 30*time.Second),
-			Transport: http.DefaultTransport,
-		},
+		HTTPTimeout:        env.GetOrDefaultSecond(EnvHTTPTimeout, 30*time.Second),
```

Comment:
```suggestion
		HTTPTimeout:        env.GetOrDefaultSecond(EnvHTTPTimeout, 60*time.Second),
```
https://github.com/exoscale/egoscale/blob/93dd01ff4407ac730efc1e08d0e664c54770a690/v2/client.go#L19